.jenkins/cscs-perftests/batch.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash -l
+
+# Copyright (c) 2020 ETH Zurich
+#
+# SPDX-License-Identifier: BSL-1.0
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+# Computes the status of the job and store the artifacts
+status_computation_and_artifacts_storage() {
+    ctest_exit_code=$?
+    ctest_status=$(( ctest_exit_code + configure_build_errors + test_errors + plot_errors ))
+
+    # Copy the testing directory for saving as an artifact
+    cp -r ${build_dir}/Testing ${orig_src_dir}/${configuration_name}-Testing
+    cp -r ${build_dir}/reports ${orig_src_dir}/${configuration_name}-reports
+
+    echo "${ctest_status}" > "jenkins-hpx-${configuration_name}-ctest-status.txt"
+    exit $ctest_status
+}
+
+# Args for the pyutils suite
+logfile=jenkins-hpx-${configuration_name}.out
+
+orig_src_dir="$(pwd)"
+src_dir="/dev/shm/hpx/src"
+build_dir="/dev/shm/hpx/build"
+
+# Copy source directory to /dev/shm for faster builds
+mkdir -p "${build_dir}"
+cp -r "${orig_src_dir}" "${src_dir}"
+envfile=${src_dir}/.jenkins/cscs-perftests/env-${configuration_name}.sh
+# Copy the perftest utility in the build dir
+mkdir -p ${build_dir}/tools/perftests_ci
+cp -r ${src_dir}/tools/perftests_ci/* ${build_dir}/tools/perftests_ci
+
+# Variables
+perftests_dir=${build_dir}/tools/perftests_ci
+result_dir=${perftests_dir}/results
+mkdir -p $result_dir
+result=$result_dir/local-priority-fifo.json
+mkdir -p ${build_dir}/reports
+
+# Load python packages
+source /apps/daint/SSL/HPX/virtual_envs/perftests_env/bin/activate
+
+# FIXME: we can probably do this step as the other tests and use gridtools
+# script only for running and plotting
```

Comment:
Still for this PR, or not necessary anymore?

Reply:
I have to try it out still